atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -354,7 +354,7 @@ def get_logs(self):
         docker_logs = NamedTemporaryFile(prefix="docker-%s" % self.build_id,
                                          suffix=".log",
                                          mode='wb')
-        docker_logs.write("\n".join(self.workflow.build_result.logs).encode('UTF-8'))
+        docker_logs.write(b"\n".join(self.workflow.build_result.logs))
```

Comment:
This value comes from atomic-reactor itself and wouldn't have changed type I think.